server.py

```diff
@@ -14,18 +14,15 @@
 from db import Code, LoginCode, PendingEmail, User
 import util
 
+config = util.loadYaml('config.yaml')
 
 CODE_VALIDATOR = re.compile(r'^(\w{8})$')
 CODE_SIZE = 8
 LOGIN_COOKIE_SIZE = 16
 
-NAME = 'Funbox'
-
-app = Flask('Funbox Accounts')
+app = Flask(config['naming']['account_service'])
```

Comment:
I'm not married to calling this app `Funbox Accounts`. Let's just use a single config field `service_name: 'Funbox'` and use that name for emails and the app name.